core/server/master/src/main/java/alluxio/master/meta/MetaMasterSync.java

```diff
@@ -56,32 +54,22 @@
   /** Last System.currentTimeMillis() timestamp when a heartbeat successfully completed. */
   private long mLastSuccessfulHeartbeatMs;
 
+  /** The ID of this standby master. */
+  private AtomicReference<Long> mMasterId = new AtomicReference<>(-1L);
+
   /**
    * Creates a new instance of {@link MetaMasterSync}.
    *
-   * @param masterId the master id
    * @param masterAddress the master address
    * @param masterClient the meta master client
    */
-  public MetaMasterSync(AtomicReference<Long>  masterId,
-      Address masterAddress, MetaMasterMasterClient masterClient) throws IOException {
+  public MetaMasterSync(Address masterAddress, MetaMasterMasterClient masterClient)
+      throws IOException {
```

Comment:
do we still need to throw exception here? if not, you can remove the next TODO

Reply:
We do not need throw exception here and I deleted both the throws statement and TODO